clifford/test/test_multivector_inverse.py

```diff
@@ -6,16 +6,38 @@
 
 class TestClosedForm:
 
+    @pytest.mark.parametrize('r', range(2))
     @pytest.mark.parametrize('p, q', [
         (p, total_dims - p)
         for total_dims in [1, 2, 3, 4, 5]
         for p in range(total_dims + 1)
     ])
```

Comment:
```suggestion
    @pytest.mark.parametrize('p, q, r', [
        (p, p_add_q - p, r)
        for p_add_q in [1, 2, 3, 4, 5]
        for p in range(p_add_q + 1)
        for r in range(2)
        if r + p_add_q <= 5  # beyond this the hitzer_inverse is not supported
    ])
```
and remove the `xfail` below